README.md

```diff
@@ -118,3 +118,26 @@ Additionally, some globals have been added to the testing environment to streaml
 * `expect` - Corresponds with `require('chai').expect`. [Info](http://chaijs.com/api/bdd/)
 * `faker` - Corresponds with `require('faker')`. [Info](https://github.com/marak/Faker.js/)
 * `sandbox` - Corresponds with `require('sinon').sandbox`. Should be used anytime when wanting to create a `sinon.spy` or `sinon.stub` as it can be easily used to reset/restore multiple spys and stubs. [Info](http://sinonjs.org/releases/v4.1.6/sandbox/)
+
+## Contributing/Publishing
+
+There are certain steps that should be taken when contributing and publishing a new release of the `contxt-sdk`.
+
+### Contributing
+
+Be sure to run `npm run build` before putting your pull request up for code review. While this is command is run in
```

Comment:
What do you think of putting `npm run build:docs` here instead of the full build? The built JS files don't get pushed GitHub, but at the same time, it's less characters and might help people if they had done `npm link`

Reply:
I've changed this paragraph a little bit to try and make it more direct. Let me know what you think